### PR TITLE
Bicycle Shop

### PR DESCRIPTION
**Pull request by DanyloSemenov**
[DEMO LINK](https://DanyloSemenov.github.io/layout_miami/

### CODE REVIEW

**Review by mbruhler — CHANGES_REQUESTED** [review]
Hi,

I don't know what is it - maybe a test PR - but for now your layout is completely off, please refer to the mockup in order to fix it

Good luck!

**Review by mvjl000 — CHANGES_REQUESTED** [review]
1.  Take the streets text on the hero section is way too small.
2. Open menu button on the hero page should be a 'burger'. Only when the menu is opened it should turn into X.
Screenshot 2023-10-23 at 08 26 39
3. There should be a phone number visible on phone icon hover
4. The mybike logo is to small
5. A phone number in the menu should be 'call-able' a link (`href="tel:"`)
6. Your menu needs more padding at the bottom
7. On a bigger resolutions those should be in the same row
Screenshot 2023-10-23 at 08 33 58
Basically you need to copmare your solution to figma very closely, adjust it and you are gonna be on the right track!

**Comment by DanyloSemenov** [comment]
I'm so sorry I didn't make commits, I'll correct myself :)

**Review by JoyCoffeeAddict — CHANGES_REQUESTED** [review]
1. icons have wrong size
image
1. image here has too much high
image
1. Also image above is in low resolution, please use the script that Paweł showed on today's lecture.
1. Email in address is not working, take a look [here](https://css-tricks.com/snippets/html/mailto-links/)
1. Close button in the menu should be exactly in the same spot as open button
Animation
1. Links in the menu are not working


Good job overall

**Comment by DanyloSemenov** [comment]
and again im remember to make a commits, sorry.
1. corrected icon size
2. added a script for images
3. added email adress 
4. corrected button in the menu
5. corrected links in the menu

**Review by mvjl000 — CHANGES_REQUESTED** [review]
1. The hero section shouldn't take more than 100vh,
2. Buttons are missing a hover effect (btw. the send button has some default border),
3. Clicking on the logo gives me 404, it shouldn't be that way (checklist points 5, 6)

**Review by mbruhler — CHANGES_REQUESTED** [review]
Looks great by now! 
One thing I can see is the last bottom photo really blurry. Please fix it.

**Review by mbruhler — CHANGES_REQUESTED** [review]
I've found one more issue: 
Other images are also blurry. Please check if you use proper size for the viewport
image
image

but check also other images

**Review by mvjl000 — CHANGES_REQUESTED** [review]
In the figma (on mobile), we can see the entire front wheel. You should reposition it a bit to the left. Also the resolution of the photo is not ideal.
Screenshot 2023-10-26 at 09 25 00

**Review by JoyCoffeeAddict — CHANGES_REQUESTED** [review]
1. Margins and padding on mobile here are a little bit off, so if you open the page on the smallest phone, you can't see the whole menu 
image

I think it's the last issue.

**Review by JoyCoffeeAddict — APPROVED** [review]
